draft-packages/hierarchical-menu/KaizenDraft/HierarchicalMenu/styles.module.scss

```diff
@@ -2,6 +2,7 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/animation";
 @import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/shadow";
 @import "~@kaizen/deprecated-component-library-helpers/styles/layout";
```

Comment:
Worth removing/replacing the deprecated-component-library-helpers files to use tokens?

Reply:
Saw your comment on slack - Approved this PR for the purpose of updating border colours.

Reply:
Yeah, had a quick look but there's a lot of legacy stuff here, that would be a larger piece of work.